common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -210,20 +211,44 @@ function (Iterator) {
          * not differs from current speed.
          */
         setSpeed: function (speed, silent, forceUpdate) {
-            if (speed !== this.currentSpeed || forceUpdate) {
+            if (speed !== this.state.speed || forceUpdate) {
                 this.speedsContainer
                     .find('li')
-                    .removeClass('is-active')
-                    .siblings("li[data-speed='" + speed + "']")
-                    .addClass('is-active');
+                    .siblings("li[data-speed='" + speed + "']");
 
-                this.speedButton.find('.value').html(speed + 'x');
+                this.speedButton.find('.value').text(speed + 'x');
                 this.currentSpeed = speed;
-
+                this.state.speed = speed;
```

Comment:
Why we're setting `speed` to `this.state.speed` when it's already present on `this.currentSpeed` ?